frontend/apps/reader/modules/readerback.lua

```diff
@@ -73,8 +76,24 @@ function ReaderBack:onBack()
             return true
         end
     else
-        logger.dbg("[ReaderBack] no location history, closing")
-        self.ui:handleEvent(Event:new("Close"))
```

Comment:
This line should be replaced with something like `self.ui:onHome()` to implement the discussed behavior.

Reply:
Yes, that's what I wanted to do, but to do it properly, I was thinking of respecting the user choice "Start with" to be the homepage. And that call for a little bit more work.

Reply:
Well yes, the proper fix can wait but I'd rather have the smaller diff combined with the intended default behavior. ;-)